itou/templates/apply/includes/job_applications_filters.html

```diff
@@ -37,13 +37,18 @@ <h5 class="mt-4">{% trans "Statut" %}</h5>
                       </div>
                       {% endfor %}
 
-                    {% if filters_form.sender %}
+                    {% if filters_form.senders %}
                         <h5 class="mt-4">{% trans "Emetteur" %}</h5>
-                          {% bootstrap_field filters_form.sender form_group_class="form_group m-3" %}
+                          {% bootstrap_field filters_form.senders form_group_class="form_group m-3" %}
                     {% endif %}
 
-                    {% if filters_form.sender_organization %}
-                          {% bootstrap_field filters_form.sender_organization form_group_class="form_group m-3" %}
+                    {% if filters_form.sender_organizations %}
+                          {% bootstrap_field filters_form.sender_organizations form_group_class="form_group m-3" %}
+                    {% endif %}
+
+                    {% if filters_form.job_seekers %}
+                        <h5 class="mt-4">{{ filters_form.job_seekers.label }}</h5>
```

Comment:
TIL `mt-4` means `set margin-top to 4 spacing units` (`{property}{sides}-{size}`) in bootstrap. https://stackoverflow.com/questions/41574776/what-is-class-mb-0-in-bootstrap-4  https://getbootstrap.com/docs/4.0/utilities/spacing/